tools/scoring/score/dimensions/dimension.py

```diff
@@ -11,71 +11,103 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Core functionality parent class"""
+"""Core component base class"""
 
-from dataclasses import dataclass
+from typing import Any, Dict, List, Tuple
 
 
-@dataclass
 class Dimension:
   """
     Container for floating-point results which
     quantify success in each dimension
 
     Attributes:
-      correct_virtual: Number of attempts achieved within virtual devices
-      correct_reporting: Number of attempts achieved within reporting devices
-      correct_ceiling_virtual: Number of attempts possible to achieve
+      translations: Proposed and solution translations
+        for all matched reporting entities. Assigned via argument
+      deserialized_files: Parsed configuration files.  Assigned via argument
```

Comment:
what is a deserialized file ? do you have a list of entity instance?

Reply:
Exactly that: `deserialized_files` is the repackaged output of `validator.Deserialize()` (parse_config.py:55). I added more complete type annotation to `__init__()` definition below this. (I was holding off since I'm focusing on the less complex dimensions ATM.)

Reply:
can you add more details in the docstring about its content what it is inside it what kind of objects it has in it.
otherwise deseralized is too vague

Reply:
Added some detail but I'm not sure how to be a lot more specific without poorly rehashing what will be specified with the type.